proposal/psychicalcoder/README.org

```diff
@@ -0,0 +1,66 @@
+#+TITLE: Mesh Simplification Using Quadric Error Metrics
+
+* Basic Information
+  
+  The GitHub repository: TODO
+
+* Problem To Solve
+
+  3D models nowadays can be very detailed and complicated. However rendering
+  these models requires a lot of computing resources, espically when there are
+  multiple light sources. In many scenarios, there is no need to use such
+  realistic models. Users can simplify their complicate model without intensively
+  breaking the contour of the object.
+
+  The algorithm I would like to use is [[http:www.cs.cmu.edu/~garland/Papers/quadrics.pdf][Surface Simplification Using Quadric Error Metrics]]
+  published by Garland and Heckbert in 1997. This method is an iterative edge
+  contraction algorithm that makes it easy to set up different simplification
+  level. In brief, we gradually replace two close vertices with a new one. The
+  new position is chosen by minimizing the total square distance between it
+  and the planes that connect to the original two vertices. Since the square
+  distance is quadratic, the new point can be found by differentiating and
+  solving a linear system. 
+
+* Perspective users
+
+  + For those who want to create low-poly art, they can easily generate one with
+    an existing 3d model.
+  + 3D game developers can reduce the computer load by
+    replacing some less important models with simplier ones.
+  
+* System Architecture
+
+  [[./architecture.png]]
```

Comment:
Can you describe how do you plan to design the data structure for your mesh?

While I think you are dealing with surface mesh, it will be interesting to think about how to work with body mesh, and body mesh of non-simplex mixed elements.